lms/static/sass/discussion/_discussion.scss

```diff
@@ -308,9 +308,29 @@ body.discussion {
     }
   }
 
-  .new-post-btn {
-    @include float(right);
-    @include margin-right(4px);
+  .inline-discussion-topic {
+    width: flex-grid(12);
+    font-size: $forum-small-font-size;
+    .inline-discussion-topic__title {
```

Comment:
Also AFAIK this double-underscore is not a pattern we use anywhere else - it also mixes dashes and underscores. Class names should be kebab cased.